math/stringGlues.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+#include <string>
+#include <vector>
+using namspace std;
+
+string stringGlues(vector<string> v) {
```

Comment:
Follow our style guidelines: this should be `StringGlues`.

Add a function comment explaining what this function does, what its parameters are, and what its return value is.